src/main/java/picocli/CommandLine.java

```diff
@@ -3746,22 +3746,12 @@ private static String format(String formatString, Object... params) {
          * </p>
          * <b>A note on boolean options</b>
          * <p>
-         * By default picocli does not expect boolean options (also called "flags" or "switches") to have a parameter.
-         * You can make a boolean option take a required parameter by annotating your field with {@code arity="1"}.
+         * By default picocli allows boolean options (also called "flags" or "switches") to have an optional parameter.
```

Comment:
Small addition:

> (...) an optional parameter, which must be either `true` or `false`.

Reply:
I think this one is very redundant, it's a _boolean_ option.

Reply:
I see what you are saying, but I think it makes sense to be very specific about what parameters are allowed. For example, the following parameters would be rejected: TRUE, FALSE, True, False, 1, 0, YES, NO, Yes, No. 

Contrast this to some languages like Groovy and Python where Strings with these values _are_ translated to true/false boolean values.